.travis/build/Dockerfile

```diff
@@ -57,18 +57,20 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	&& rm -rf /var/lib/apt/lists/*
 
 RUN { \
-		echo "mariadb-server-10.5" mysql-server/root_password password 'unused'; \
-		echo "mariadb-server-10.5" mysql-server/root_password_again password 'unused'; \
+		echo "mariadb-server-10.6" mysql-server/root_password password 'unused'; \
```

Comment:
Can't we parametrize here and later, especially in wget call, the server version?

Reply:
there has been change each time i've change server. I don't think that really can be parameterized